<ui version="4.0" >
 <class>ColorSettingsPage</class>
 <widget class="QWidget" name="ColorSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Color Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="labelBackground" >
     <property name="text" >
      <string>Background Color</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="ColorWidget" native="1" name="widgetBackground" />
     </item>
     <item>
      <widget class="QToolButton" name="buttonBackground" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelGradient" >
     <property name="text" >
      <string>Color Gradient</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="ColorWidget" native="1" name="widgetGradient" />
     </item>
     <item>
      <widget class="QToolButton" name="buttonGradient" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelOptions" >
     <property name="text" >
      <string>Gradient Options</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMirrored" >
       <property name="text" >
        <string>Mirrored</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkReversed" >
       <property name="text" >
        <string>Reversed</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelScale" >
     <property name="text" >
      <string>Mapping Scale</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DoubleSlider" name="sliderScale" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelOffset" >
     <property name="text" >
      <string>Mapping Offset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DoubleSlider" name="sliderOffset" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition" >
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval" >
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDefault" >
     <property name="text" >
      <string>Default Settings</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestore" >
       <property name="text" >
        <string>Restore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStore" >
       <property name="text" >
        <string>Store</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QWidget</extends>
   <header>colorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DoubleSlider</class>
   <extends>QSlider</extends>
   <header>doubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
